cmake/KPXCHelpers.cmake

```diff
@@ -1,33 +1,61 @@
-function(kpxc_run_macdeployqt_on_installed_helper_binary app_name installed_binary_relative_path)
-    # Running macdeployqt on a POST_BUILD copied binary is not useful for CPack, because
-    # the copied binary will be overriden by the corresponding install(TARGETS) command of the same
-    # binary.
-    # Thus, we run macdeployqt using install(CODE) on the already installed binary.
-    # The precondition is that install(TARGETS) has to be called before this function is called.
+# Running macdeployqt on a POST_BUILD copied binaries is pointless when using CPack because
```

Comment:
I think it makes sense to give this file a more specific name. No need to prematurely abstract things.

Reply:
Done